### ISSUE TITLE
TypeError in `profiler.build_suite()` during Expectation Suite initialization by profiling while parsing timestamp value

### ISSUE THREAD

**Issue by riwim** [fivetran/great_expectations#5421]
**Describe the bug**
I have a BigQuery table which cannot be parsed with the profiler. I was able to reduce this table to one row and one column: The value is a timestamp field with the content `2022-05-25 13:27:30.456000+00:00`.
![image](https://user-images.githubusercontent.com/18545483/176790130-b7f321e7-833e-4f3b-b541-2191acb5b390.png)

When this line is executed in the initialization notebook

```python
suite = profiler.build_suite()
```

then I get this TypeError:

```
File ~/ge/.venv/lib/python3.9/site-packages/great_expectations/expectations/expectation.py:1705, in TableExpectation._validate_metric_value_between(self, metric_name, configuration, metrics, runtime_configuration, execution_engine)
   1702         except TypeError:
   1703             pass
-> 1705 if not isinstance(metric_value, datetime.datetime) and np.isnan(metric_value):
   1706     return {"success": False, "result": {"observed_value": None}}
   1708 # Checking if mean lies between thresholds

TypeError: ufunc 'isnan' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''
```

**To Reproduce**
1. Same steps as those to reproduce #5420, plus
2. Comment out the only field in the list _exclude_column_names_ (in my case _derived_tstamp_)
3. Execute all code including `suite = profiler.build_suite()`

**Expected behavior**
I would expect no TypeError to be thrown when parsing a timestamp field.

**Environment**

- Operating System: MacOS 12.3.1
- Great Expectations Version: 0.15.11

**Comment by talagluck** [comment]
Thanks for raising this, @riwim . This appears to be a bug that was introduced relatively recently. We will review and be in touch.

**Comment by alexsherstinsky** [comment]
This issue has been fixed in #5502 -- thank you for pointing it out!
